src/path.cpp

```diff
@@ -275,7 +275,7 @@ static void maybe_issue_path_warning(const wcstring &which_dir, const wcstring &
         debug(0, _(L"The error was '%s'."), strerror(saved_errno));
         debug(0, _(L"Please set $%ls to a directory where you have write access."), env_var);
     }
-    write(STDERR_FILENO, "\n", 1);
+    write_ignore(STDERR_FILENO, "\n", 1);
```

Comment:
I agree with your assessment; I did not feel it was my place as such a newcomer to suggest silencing the warnings entirely and just wanted to unify the warning suppression to use the existing `read|write_ignore` method. If we have no intention of handling write to stderr failure somehow, then it makes no sense leaving the `write_ignore` calls lying around.